packages/manager/src/components/VerticalLinearStepper/VerticalLinearStepper.styles.ts

```diff
@@ -48,6 +48,6 @@ export const StyledColorlibConnector = styled(StepConnector, {
   '& .MuiStepConnector-line': {
     borderColor: '#eaeaf0',
     borderLeftWidth: '3px',
-    minHeight: '28px',
+    minHeight: '16px',
```

Comment:
if this takes in a theme, maybe we can do 
```suggestion
    minHeight: theme.spacing(2),
```
to avoid hardcoding?